webpack.config.js

```diff
@@ -1,3 +1,4 @@
+const fs = require("fs");
```

Comment:
Nice, thank you! Can you please cherry-pick the corresponding README files as well, to show how to use this?

Reply:
Ah, I was thinking about it, but we seem to miss a lot so wasn't sure if I should have added. But as you agree I'll add it!

Reply:
Added a section of the install section of the starterkit README, as that information is in c-podman in HACKING.md